lib/modules/dosomething/dosomething_fact_page/dosomething_fact_page.module

```diff
@@ -90,22 +91,33 @@ function dosomething_fact_page_preprocess_node(&$vars) {
 
     foreach ($template_vars as $key => $labels) {
       foreach ($labels as $label) {
-        $field = "field_{$label}";
-          if (isset($content[$field])) {
+        $field = $vars["field_{$label}"];
+
+        if (isset($field) && !empty($field)) {
+          $prefix = array_keys($field)[0];
+
+          if (!is_int($prefix) && in_array($prefix, $supported_languages)) {
+            $field = $field[$prefix];
+          }
+
+          $field = $field[0];
```

Comment:
The above lines (96-103) of logic is not super pretty but gets the job done. I could extract it to a function later if it ends up we have to do similar stuff in other locations.